docs/content/discovery.md

```diff
@@ -149,8 +178,11 @@ Below is a policy file which generates an OPA congfiguration.
 package example
 
 discovery = {
-  "bundle": {
-    "name": bundle_name                # line 5
+  "bundles": {
+    "main": {
+      "service": "acmecorp", #line 5
```

Comment:
nit: the `line 5` comment is referenced below for the `bundle_name` variable. I think we need to update it to `line 6` now

Reply:
fixed - thanks!

Reply:
Close, but we need to move the comment down to line 6 and update the paragraph below the snippet. As is its saying:

```The `bundle_name` variable in `line 5` of the above policy will be dynamically selected based on the value of the label `region`.```

Which isn't accurate for the example.

Reply:
Fixed, I think it should be line 7 if I count correctly.